client-app/src/desktop/common/grid/SampleGridModel.ts

```diff
@@ -117,7 +117,7 @@ export class SampleGridModel extends HoistModel {
                 return a < b ? -1 : 1;
             }
         },
-        restoreDefaultsFn: () => this.restoreDefaultsFn(),
+        restoreDefaultsFn: async () => this.restoreDefaultsFn(),
```

Comment:
is this needed?

Reply:
Not needed! Think this came from before our change last week